core/src/main/java/com/crazylegend/core/dto/Config.kt

```diff
@@ -0,0 +1,3 @@
+package com.crazylegend.core.dto
+
+data class Config(val showFileSize: Boolean = false)
```

Comment:
This isn't necessary, please use the configuration as before through the builder

Reply:
done